velox/benchmarks/basic/LikeFunctionsBenchmark.cpp

```diff
@@ -233,7 +233,7 @@ BENCHMARK(tpchQuery20) {
 } // namespace
 
 int main(int argc, char* argv[]) {
-  folly::init(&argc, &argv, true);
```

Comment:
@assignUser the new benchmark issue we are seeing is due to this change. https://github.com/facebookincubator/velox/actions/runs/7108997656/job/19353240377?pr=7700#step:14:150

Reply:
I wonder why the RAII version is causing failures. 
I will try to investigate if there is something fundamental lurking around.